glossary.dd

```diff
@@ -81,6 +81,14 @@ $(DL
 	maximize the portability of code.
 	)
 
+	$(DT $(LNAME2 lvalue, lvalue))
+	$(DD An lvalue is an abstract term referring to a value contained within a
+	variable (including elements of arrays and associative arrays).
+	Characteristics of lvalues are they have a memory address, they can be
+	assigned new values and can be passed to (or be returned from) functions by
+	reference. An lvalue is the counterpart to an rvalue.
+	)
+
```

Comment:
Yah, "contained within a variable" is imprecise because it doesn't cover expressions yielding lvalues and things like immutable data. Proposed text:

```
$(DT $(LNAME2 lvalue, lvalue))
$(DD An lvalue is an abstract term referring to a value with an accessible address (through e.g. the unary $(D &) operator). Typical examples of lvalues include variables, constants introduced with `const` or `immutable` (but not `enum`), and elements of arrays and associative arrays. Calls to functions that return `ref` and pointer dereference expressions are also lvalues. Lvalues can be passed to (or be returned from) functions by
reference. An lvalue is the counterpart to an rvalue.
)
```